core/color.h

```diff
@@ -210,6 +210,16 @@ struct Color {
 		a = 1.0;
 	}
 
+	/**
+	 * Copy constructor
+	 */
+	_FORCE_INLINE_ Color(const Color &c2) {
```

Comment:
Why is everything `c2` etc? It'd be better to call it `p_c` etc.

Reply:
Sure thing, I'll fix that.